build.rs

```diff
@@ -1,8 +1,39 @@
-extern crate plume_common;
```

Comment:
You should probably remove it from `Cargo.toml` too.

Reply:
I totally forgot about that